index.js

```diff
@@ -56,42 +60,54 @@ const zooAnimals = [
   displayNames will be an array of strings, and each string should follow this pattern: "name: {name}, scientific: {scientific name}"
   */
 
-  function animalNames(/*Your Code Here*/){
-    /*Your Code Here*/
+  function animalNames(arr){
+    let names = [];
+    arr.forEach(element => {
+        names.push(`name: ${element.animal_name}, scientific: ${element.scientific_name}`);
+    });
+    return names;
   }
-  
+  console.log(animalNames(zooAnimals));
 
   /* 🦁🦁🦁 Request 2: .map() 🦁🦁🦁
   The zoo needs a list of all their animal's names converted to lower case. 
   Using lowerCaseNames use .map() to create a new array of strings with the animal's names in lowercase and return the new array. 
   For example: ['jackal, asiatic', .....]
   */
 
-  function lowerCaseNames(/*Your Code Here*/){
-    /*Your Code Here*/
+  function lowerCaseNames(arr){
+    let nArr = [];
+    arr.map(el => {
+      nArr.push(el.animal_name.toLowerCase());
+    });
```

Comment:
you should return `el.animal_name.toLowerCase()` here and let map handle adding that to your array when you set `nArr` to be the result of calling `arr.map`